docs/meshcloud.policies.md

```diff
@@ -0,0 +1,64 @@
+---
+id: meshcloud.policies
+title: meshPolicies
+---
+
+## Introduction: what are meshPolicies?
+
+A meshPolicy is a set of rule(s) between two [meshObjects](meshcloud.index.md#introduction) that are globally defined by your organization. Each rule describes what [tags](meshcloud.tag-schema.md) have to match on each meshObject, in order to comply with the meshPolicy. meshPolicies are enforced in various places when attaching meshObjects.
```

Comment:
Can we add an explanation what "match" means in this context. Something like the tags of one objects are a complete match or a subset of the tags of the comparing objects.

Reply:
Removed the word match and replaced it with a more concrete explanation. I don't think this is the place to expand on subset tag matching, also because we don't even support this yet.